parts/language/manual_ja.txt

```diff
@@ -13,50 +13,88 @@
 スピンの検出:
         「3コーナ」ルールを満たした時 +2点
         「インモービル (Immobile)」 ルールを満たした時 +2点
-        - 上のどちらかを満たした時, スピンが確定します。
+        - 上のどちらかを満たした時、スピンが確定します。
 
-        もし回転システムがキックテーブルから2番チェック (2nd check) を採用していなかった場合 +1点
-        - もしスピンの合計点が2点しかない場合は「ミニ」扱いになります；ミノがS, Z, J, L, Tのいずれかで, ラインクリア時にそのミノを全て消せなかった場合のみ。
+        もし回転システムがキックテーブルから2番目のチェック (2nd check) を採用していなかった場合 +1点
+        - もしスピンの合計点が2点しかない場合は「ミニ」扱いになります。ただしこれは、ミノがS,Z,J,L,Tのいずれかで、line消去の時にそのミノのブロックが全部消せなかった場合のみ。
 
-Attack system:
-        [TODO]
-Back to Back (B2B) gauge:
-        [TODO]
+攻撃システム:
+        Special clear:
+                Special clearとは、spin、techrash(もしくは一度に4段以上のline消去)、PC、HPCのどれかを行うことです。Special clearを行うとB2Bゲージが上昇します。
+        Super clear:
+                Super clearとは、B2Bゲージがある状態でSpecial clearを行うことです。
+        Special clears (spin):
+                消したline数の2倍の火力を送ります、
+                - B2Bは、spin single/doubleで1line、spin tripleで2line、techrashで4line、techrash+で8lineを追加で送ります。
```

Comment:
If we use "spin single/double/triple", we should also use "spin techrash" and "spin techrash+" here.